template.go

```diff
@@ -32,10 +32,12 @@ import (
 var (
 	beegoTplFuncMap = make(template.FuncMap)
 	// beeTemplates caching map and supported template file extensions.
-	beeTemplates  = make(map[string]*template.Template)
+	beeTemplates  = make(map[string]TemplateI)
```

Comment:
interface  in golang is named with -er ,such as io.Writ**er**,io.Read**er**,http.Flush**er**